example_walk_through_test.go

```diff
@@ -0,0 +1,25 @@
+// Run Tarantool instance before example execution:
+// Terminal 1:
+// $ TT_LISTEN=3013 TT_WORK_DIR=$(mktemp -d -t 'tarantool.XXX') tarantool config.lua
+//
+// Terminal 2:
+// $ go test -v example_test.go
+package tarantool_test
+
+import (
+	"fmt"
+	"github.com/tarantool/go-tarantool"
+)
+
```

Comment:
It would be nice to have a sentence or to about what this example is. Seeing this block of code in examples section is a bit confusing
![image](https://user-images.githubusercontent.com/20455996/163340939-426a6284-117b-44f8-864b-766020f56a28.png)

Reply:
I would remove it at all. Now we have examples for `Connect`, `Insert` and so on. This example looks useless in tests. Do you remember that we also have the same example in a README with comments?

Reply:
Removed Walking Through compilable example.